src/gui/windows/graphics.zig

```diff
@@ -74,6 +74,15 @@ fn fovFormatter(allocator: main.utils.NeverFailingAllocator, value: f32) []const
 	return std.fmt.allocPrint(allocator.allocator, "#ffffffField Of View: {d:.0}°", .{value}) catch unreachable;
 }
 
+fn fovSpeedAddCallback(newValue: f32) void {
+	settings.speedFovAdd = newValue;
+	settings.save();
+}
+
+fn fovSpeedAddFormatter(allocator: main.utils.NeverFailingAllocator, value: f32) []const u8 {
+	return std.fmt.allocPrint(allocator.allocator, "#ffffffFoV Speed Widening: {d:.0}°", .{value}) catch unreachable;
+}
```

Comment:
I'm not happy with the naming here. How do other games call this option?

Also the naming is inconsistent: The function is called `fovSpeedAdd`, the option is called `speedFovAdd` and the user sees `FoV Speed Widening`.

Reply:
> I'm not happy with the naming here. How do other games call this option?
> 
> Also the naming is inconsistent: The function is called `fovSpeedAdd`, the option is called `speedFovAdd` and the user sees `FoV Speed Widening`.

Good point. I'll check what other games do as nothing comes to mind right now. 

Not sure why I skipped over the naming inconsistencies. Will fix, thanks!

What do you think of the points brought up by Ikabod-kee? I imagine you have the final say, right? :D